aries-site/src/pages/tokens/using-tokens-in-figma.mdx

```diff
@@ -0,0 +1,45 @@
+import { Notification } from 'grommet';
+import { TokenTypes } from '../../examples';
+
+<Notification
+  message="Design tokens as Figma variables are currently in alpha testing. It's possible that variable names will change in v1 resulting in unlinked styles."
+  width={{ max: 'large' }}
+  margin={{ bottom: 'medium' }}
+  status="warning"
+/>
+
```

Comment:
Like in our talk this morning @oliverHPE I believe we should be a bit more specific to whom we are talking to. Like in "Using tokens in code", I would suggest we have a section explaining that if you use the HPE DS components, there's no need to consume the component collection. I would also clarify that the product designers can use the semantic collection for layouts and such, but should not edit components coming from the official library.
I would then add a section for designers that maintain other libraries in HPE or even designers who want to try creating new components to suggest to the HPE Design System

Reply:
I'll update this with any new drafts you have Oliver and once we've ironed out where we'll house things.